### PR TITLE
Make handle a parameter in the PM template

### PR DESCRIPTION
**Pull request by zachthompson**
Allow user to choose their handle during IA creation.

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by russellholt** [comment]
LGTM

**Comment by GuiltyDolphin** [comment]
@zachthompson This has conflicts - you sorted this elsewhere?

**Comment by zachthompson** [comment]
@GuiltyDolphin I'll check it out

**Comment by zachthompson** [comment]
Should be good now.  There was conflict in the [duckpan pr](https://github.com/duckduckgo/p5-app-duckpan/pull/304) as well.

**Comment by GuiltyDolphin** [comment]
@zachthompson Looks all sorted now :+1:

File: template/lib/DDG/Spice/Example.pm [hunk 1]
```diff
@@ -16,14 +16,15 @@ spice wrap_jsonp_callback => 0; # only enable for non-JSONP APIs (i.e. no &callb
...
 
 # Handle statement
-handle remainder => sub {
+handle <: $ia_handler :> => sub {
 
-    # Query is in $_...if you need to do something with it before returning
-    return $_;
+    # Query is in $_ or @_, depending on the handle you chose...if you
+    # need to do something with it before returning
+    return \<: $ia_handler_var :>_;
```
Comment (moollaza) [inline template/lib/DDG/Spice/Example.pm]: @zachthompson was the `\` intentional?

It creates `\$_` which seems unnecessary?

**Pull request merged**